api/app/kobo.py

```diff
@@ -227,18 +218,18 @@
 def get_form_responses(
     begin_datetime: datetime,
     end_datetime: datetime,
-    form_name: str,
+    form_id: str,
     datetime_field: str,
     geom_field: str | None,
     filters: str | None,
     form_url: str,
     province: str | None = None,
```

Comment:
Should not be optional since the form id is **always** needed to get the form data and metadata. If the form id is invalid the kobo api will return a 404 which is handled by the _raise_for_status_ method

Reply:
I think Graham was referring to the typings below :)

Reply:
thanks @ericboucher. I guess nothing I can do here since the project is written based using this syntax